lib/compat/wordpress-6.9/block-bindings.php

```diff
@@ -112,10 +112,12 @@ function gutenberg_block_bindings_render_block( $block_content, $block, $instanc
 function gutenberg_get_block_bindings_supported_attributes( $block_type ) {
 	// List of block attributes supported by Block Bindings in WP 6.8.
```

Comment:
@ockham There's some references to 6.8. Can I get a confidence check on this file 🙏